marketplace_builder/views/pages/tutorials/data-import-export/import.liquid

```diff
@@ -0,0 +1,104 @@
+---
+converter: markdown
+metadata:
+  title: Data Import
+  description: Import data
+slug: tutorials/data-import-export/import
+searchable: true
+---
+For now you can export/import instance data on new stack. Data to be imported needs to be in json format:
+
+```json
+{
+  "users": [...],
+  "models": [...],
+  "transactables": [...]
+}
+```
+
+You can generate data from existing instance with `marketplace-kit data export` command([more details here](/tutorials/data-import-export/export#cli-export)). It will generate `data.json` file. You can also import data from other sources(old stack, other systems) once you generate proper json file. This is fairly easily to convert for example csv file into json file.
```

Comment:
```suggestion
You can generate data from existing instance with `marketplace-kit data export` command([more details here](/tutorials/data-import-export/export#cli-export)). It will generate `data.json` file. You can also import data from other sources(old stack, other systems) once you generate proper JSON file. This is fairly easily to convert for example csv file into json file.
```